lib/manager/gradle/index.spec.ts

```diff
@@ -14,26 +12,38 @@ import * as _util from '../../util';
 import { ifSystemSupportsGradle } from './__testutil__/gradle';
 import * as _manager from '.';
 import { ExtractConfig } from '../common';
+import { GRADLE_DEPENDENCY_REPORT_FILENAME } from './gradle-updates-report';
 
 const fixtures = 'lib/manager/gradle/__fixtures__';
-
-const config = {
-  localDir: 'localDir',
+const standardUpdatesReport = () =>
+  fs.readFile(`${fixtures}/updatesReport.json`, 'utf8');
+const emptyUpdatesReport = () =>
+  fs.readFile(`${fixtures}/updatesReportEmpty.json`, 'utf8');
+const multiProjectUpdatesReport = () =>
+  fs.readFile(`${fixtures}/MultiProjectUpdatesReport.json`, 'utf8');
+
+const baseConfig = {
   gradle: {
     timeout: 60,
   },
 };
 
-const updatesDependenciesReport = fsReal.readFileSync(
-  'lib/manager/gradle/__fixtures__/updatesReport.json',
-  'utf8'
-);
-
 const gradleOutput = {
   stdout: 'gradle output',
   stderr: '',
 };
 
+const replacingSerializer = (
+  search: string,
+  replacement: string
+): jest.SnapshotSerializerPlugin => ({
+  test: value => typeof value === 'string' && value.includes(search),
+  serialize: (val, config, indent, depth, refs, printer) => {
+    const replaced = (val as string).replace(search, replacement);
+    return printer(replaced, config, indent, depth, refs);
+  },
+});
+
```

Comment:
This is a great idea. Could you extract it to `test/utils.ts`, as there are quite a few places we mock `fs` to get consistent path names, and this is a much better approach.